### DISCUSSION TITLE
Videos not download in Firefox or Chromium. Linux Mint.

### DISCUSSION THREAD

**Discussion by SyncroScales** [subhra74/xdm#1099] [General]
Hello,

1. Videos not download with Firefox, Chromium.

Linux Mint xfce 20.2

XDM 8.0.26 BETA

Browser monitoring on.
Extensions added. Developer mode on for Chromium.

New - Video download - Find
"No video found. However there might be other ways to download the video. Please click "Learn More" button for details.

Media Grabber
Shows nothing

A window sometimes shows up, but only e.g.: watch.html is downloadable.
Address
File
Save in
Download Now, Download Later

Nothing is downloadable.

No log. No such directory exists.

The two methods on this page do not work. https://github.com/subhra74/xdm/wiki/How-to-download-streaming-video

Anything to do?

2. I noticed that the panel changed and the applications menu changed. I am not sure if XDM did this. I read some other discussions and some people have mentioned glitches. I cannot move XDM icon on the panel to right or left, the date/time/network icon/volume icon/etc shift to the center of the panel, unless applications are open. The more that are open the further it moves to where it should be locked into place. And in the applications menu now everything is small icons not a list.
Not sure if this is XDM related. These glitches started to happen  after the installation of the .deb file.

Also put into Issues #1092. Can a moderator delete that message?

Thanks.

**Comment by gunyakov** [comment]
I confirm. Firefox under Linux mint with XDM plugin cant show any pop up windows during stream video watching.
